### PR TITLE
Separate cli commands feb4

### PR DESCRIPTION
**Pull request by RichardHoch**
Replaces https://github.com/kubev2v/forklift-documentation/pull/459

### CODE REVIEW

File: documentation/modules/new-migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -453,8 +768,62 @@ EOF
 <1> You can use the default `hook-runner` image or specify a custom image. If you specify a custom image, you do not have to specify a playbook.
 <2> Optional: Base64-encoded Ansible playbook. If you specify a playbook, the `image` must be `hook-runner`.
 
+[id="test4_{context}"]
+== test
```
Comment (anarnold97) [inline documentation/modules/new-migrating-virtual-machines-cli.adoc:772]: @RichardHoch - please do not forget to change my madness

File: documentation/modules/new-migrating-virtual-machines-cli.adoc [hunk 2]
```diff
@@ -498,13 +866,167 @@ EOF
...
+<8> You can use only the `name` parameter, not the `id` parameter, to specify the source VMs.
+<9> Specify the TBD.
+<10> Optional: You can specify up to two hooks for a VM. Each hook must run during a separate migration step.
+<11> Specify the name of the `Hook` CR.
+<12> Allowed values are `PreHook`, before the migration plan starts, or `PostHook`, after the migration is complete.
+endif::[]
+
+
+[id="test5_{context}"]
+== test
```
Comment (anarnold97) [inline documentation/modules/new-migrating-virtual-machines-cli.adoc]: Yep... sorry. MADNESS!!!

Comment (RichardHoch) [reply documentation/modules/new-migrating-virtual-machines-cli.adoc]: @anarnold97 So far, your madness and my repetitions of it in the file are the only thing keeping bccutil from throwing warnings. Going to poke around a bit bit more, but I'll probably be looking for you during Office Hours.

Comment (anarnold97) [reply documentation/modules/new-migrating-virtual-machines-cli.adoc]: @RichardHoch - let me have another look later tonight

With MTA 7.0.1 and MTR 1.2.4 going out tomorrow, I might need to cancel Office Hours. But will set some time aside on Wednesday to have a look

*BUT* you have got so much good work done, chill out and give me a chance to look for the tiny problem (... and it is tiny).

File: documentation/modules/snippet_secret-cr.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+<4> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
+<5> Specify the user password.
+<6> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
+<7> {osp} only: Specify the domain name.
+<8> {osp} only: Specify the project name.
+<9> {osp} only: Specify the name of the {osp} region.
+<10> {rhv-short} and {osp} only: For {rhv-short}, enter the {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA. For {osp}, enter the CA certificate for connecting to the source environment. The certificate is not used when `insecureSkipVerify` is set to `<true>`.
```
Comment (anarnold97) [inline documentation/modules/snippet_secret-cr.adoc:49]:
```suggestion
<10> {rhv-short} and {osp} only: For {rhv-short}, enter the {manager} CA certificate unless it was replaced by a third-party certificate, in which, case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA. For {osp}, enter the CA certificate for connecting to the source environment. The certificate is not used when `insecureSkipVerify` is set to `<true>`.
```

**Pull request closed without merging**